app/models/post.rb

```diff
@@ -1,2 +1,20 @@
 class Post < ApplicationRecord
+  has_many :comments, foreign_key: :post_id
+  has_many :likes, foreign_key: :post_id
+  belongs_to :author, class_name: 'User', foreign_key: :author_id
```

Comment:
- [OPTIONAL] I think here you don't need to add again `foreign_key` as it has already been added in the user.rb file 🙃 🙃